hack/test.sh

```diff
@@ -17,4 +17,4 @@ set -e
 
 echo "> Test"
 
-GO111MODULE=on ginkgo -mod=vendor $@
+GO111MODULE=on go test -race -mod=vendor $@
```

Comment:
```suggestion
GO111MODULE=on go test -race -mod=vendor $@  | grep -v 'no test files'
```
Helps to get rid of the `[no test files]` noise.

Reply:
Good idea, done 👍